.github/workflows/checkpatch.yml

```diff
@@ -20,6 +20,8 @@ on: pull_request
 jobs:
   checkpatch:
     runs-on: ubuntu-latest
+    # skip upstream merge PR
+    if: ${{ !contains(github.head_ref, 'merge/sound-upstream') }}
```

Comment:
Will the github interface show this line explaining why it was skipped?

Reply:
I think no, maybe you can try to create an issue for GitHub actions, it is a very nice feature.